### PR TITLE
fix alt info of a picture

### PR DESCRIPTION
**Pull request by hanyujie2002**

### CODE REVIEW

**Comment by hanyujie2002** [comment]
By the way, 

https://github.com/mdn/content/blob/d532243f57ea4abddf6df23d453a869f70beddfa/files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md?plain=1#L221

This picture is not suitable

File: files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ Now the new virtual box should appear in the left-hand menu of the main Virtual
 
 After the process has completed, you should have a virtual machine running an operating system inside a window on your host computer.
 
-![Virtual box machine running on an windows operating system](virtualbox-running.png)
+![a windows operating system running on a Virtual box machine](virtualbox-running.png)
```
Comment (hamishwillee) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md:227]: Thanks - good catch. Further improved
```suggestion
![Screenshot of Windows XP running hosted in Virtual box running on macOS](virtualbox-running.png)
```

**Review by hamishwillee — APPROVED** [review]
Thanks @hanyujie2002 - much appreciated.

W.r.t. images, if one is not suitable, can you please provide a separate PR or an issue report. It would be good to, for example, also update the image for Windows XP too :-)

File: files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ Now the new virtual box should appear in the left-hand menu of the main Virtual
 
 After the process has completed, you should have a virtual machine running an operating system inside a window on your host computer.
 
-![Virtual box machine running on an windows operating system](virtualbox-running.png)
+![Screenshot of Windows XP running hosted in Virtual box running on macOS](virtualbox-running.png)
```
Comment (hamishwillee) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md:227]:
```suggestion
![Screenshot of Windows XP, hosted in Virtual box, and running on macOS](virtualbox-running.png)
```

**Pull request merged**